packages/rancher-istio-1.9/charts/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.9.6
+appVersion: 1.9.8
```

Comment:
with the dependency.yaml change below, we shouldn't need to modify the annotation for CRD install. However, if we do not go that route, you will need to bump the Chart.yaml here